docs/assembly.rst

```diff
@@ -116,30 +107,30 @@ efficient code, for example:
         }
     }
 
-.. index:: selector; of a function
 
-Access to External Variables, Functions and Libraries
------------------------------------------------------
+.. index:: bir fonksiyon seçici
+
+Dış değişkenlere, fonksiyonlara ve kütüphanelere erişim
+-------------------------------------------------------
 
-You can access Solidity variables and other identifiers by using their name.
+Solidity değişkenlerine ve diğer tanımlayıcılara isimlerini kullanarak erişebilirsiniz.
 
-Local variables of value type are directly usable in inline assembly.
-They can both be read and assigned to.
+Bir değer tipinin yerel değişkenleri inline assembly içinde kullanılabilir durumdadır.
```

Comment:
```suggestion
Bir değer tipinin yerel değişkenleri satır içi assembly içinde kullanılabilir durumdadır.
```
Bu konuda tam emin değilim "inline" kısmı çevirilmeli mi diye ama çevirilmeden de kalabilir

Reply:
Direkt olarak konu başlığı olduğu için olduğu şekilde bırakmak daha doğru geldi bana.